packages/react/src/TreeView/TreeView.tsx

```diff
@@ -459,7 +463,7 @@ const Item = React.forwardRef<HTMLElement, TreeViewItemProps>(
           id={itemId}
           role="treeitem"
           aria-labelledby={labelId}
-          aria-describedby={`${leadingVisualId} ${trailingVisualId}`}
+          aria-describedby={`${leadingActionId} ${leadingVisualId} ${trailingVisualId}`}
```

Comment:
Don't think the action should be used to describe the item, especially when the action itself is not accessible (aria-hidden + tab-index=-1) 🤔 

cc @ericwbailey for confirmation

Reply:
`aria-hidden` should hypothetically override this, but yes, I think we should remove it. 

We'll likely want to include a hint down the line in the `aria-label` about state, such as "draggable". That's something we'll be researching with TetraLogical.

Reply:
@ayy-bc Final thing for you to look at :)

Reply:
> @siddharthkp I was just following what LeadingVisual and TrailingVisual in this case. Looking at it again, I think removing this makes sense and we can add the required labels to the components itself when we pass it.
> I can remove it or you can do so, whatever you prefer (lmk).

Yes please, I think actions are different

Reply:
@siddharthkp pushed the changes